plugins/com.google.cloud.tools.eclipse.integration.appengine/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,71 @@
+cleanup.add_default_serial_version_id=true
```

Comment:
I think @akerekes means having these settings everywhere.  Yes, we should; it's part of #109